### DISCUSSION TITLE
Query regarding averaging output

### DISCUSSION THREAD

**Discussion by vijay00901** [firemodels/fds#12991] [Q&A]
Dear friends,

I have multiple FDS outputs for the same model with minor variations in initial conditions. Is it possible for me to average the output values so that I can view an averaged 3d rendering of smoke and fire in Smokeview? Please let me know. Thank you.

Vijay

**Comment by mcgratta** [comment]
If you mean that you have multiple jobs, and you want to average the output files and then view the average in Smokeview? If yes, there is no simple way to do this. You would have to read the files and do the averaging yourself.
